Error compiling libws.a

I get two errors, when calling `make all` in `c_stuff/src`. 

```
simulator.c:495:12: error: ‘off_ms’ redeclared as different kind of symbol
  495 |     cJSON *off_ms = cJSON_CreateNumber(off_ms);           // OFF part of the halfperiod
```

```
simulator.c:492:12: error: ‘on_ms’ redeclared as different kind of symbol
  492 |     cJSON *on_ms = cJSON_CreateNumber(on_ms);           // ON part of the halfperiod (PWM)
```

I fixed this by renaming the variables to `*json_off_ms` and `*json_on_ms`.

I tried compiling with gcc version 9.3.0 on Ubuntu.

--- a/c_stuff/src/simulator.c
+++ b/c_stuff/src/simulator.c
@@ -489,11 +489,11 @@ void vibrate(int count, int on_ms, int off_ms) {
     cJSON *count_out = cJSON_CreateNumber(count);             // vibration periods to execute
     cJSON_AddItemToObject(message, "periods", count_out);
 
-    cJSON *on_ms = cJSON_CreateNumber(on_ms);           // ON part of the halfperiod (PWM)
-    cJSON_AddItemToObject(message, "ms_on", on_ms);
+    cJSON *json_on_ms = cJSON_CreateNumber(on_ms);           // ON part of the halfperiod (PWM)
+    cJSON_AddItemToObject(message, "ms_on", json_on_ms);
 
-    cJSON *off_ms = cJSON_CreateNumber(off_ms);           // OFF part of the halfperiod
-    cJSON_AddItemToObject(message, "ms_off", off_ms);
+    cJSON *json_off_ms = cJSON_CreateNumber(off_ms);           // OFF part of the halfperiod
+    cJSON_AddItemToObject(message, "ms_off", json_off_ms);
 
     char *string = cJSON_Print(message);
     printf("%s\n", string);

